LDAR_Sim/src/methods/deployment/mobile_crew.py

```diff
@@ -111,20 +111,26 @@ def start_day(self, site_pool):
                         # The site order will not change if route_planning is not used
                         site_plans_today.append(site_plan)
                         est_mins_remaining -= site_plan['LDAR_mins']
-                        if est_mins_remaining <= 0:
-                            # if the day has been filled with surveys exit for and while
-                            # loop
-                            exit_flag = True
-                            break
+                        # # The following will allow the program to keep trying sites
```

Comment:
Commented out to make it the same as the old version, can reintroduce after testing. Basically it doesnt give up if one site cannot be visited if uncommented